### DISCUSSION TITLE
Difference between Array and string base input.

### DISCUSSION THREAD

**Discussion by dzolt** [joe-bell/cva#155] [Q&A] [answered]
Hey!

I've recently come across your project and I've noticed that when using cva() function you can pass either array or string value to define base class as first argument.

Does anybody know what's the difference between them? Or when would I want to actually pass a string rather than an array or other way around?

Thanks in advance!

**Comment by joe-bell** [comment, marked as answer]
It's purely for DevEx! Breaking classes into an array can sometimes be a little easier to read for longer class lists

**Reply by dzolt** [reply]
Amazing. Thank you so much!
